src/test/groovy/graphql/schema/GraphQLAppliedDirectiveArgumentTest.groovy

```diff
@@ -1,16 +1,16 @@
-package graphql
```

Comment:
Moving tests out of generically named files like "Issue 2001" and into a better home.

In this case, an Applied Directive Argument file